ban/core/versioning.py

```diff
@@ -242,6 +250,7 @@ class Diff(db.Model):
     old = db.ForeignKeyField(Version, null=True)
     # new is empty after delete.
     new = db.ForeignKeyField(Version, null=True)
+    insee = db.CharField(length=5)
```

Comment:
I'm also balanced about this field: should it be a FK pointing to Municipality instead?
I've chosen to have a flat "insee" value, to prevent dealing with hard deleting Municipality rows: if we have a PK instead, that would need either delete the Diff or allow a null value on delete.
On the other side, if we have a PK, it's easier to make the Diff endpoint evolve: we can for now only expose the "insee", but we may either replace it by the Municipality id or add some other information in the future without needing to deal with DB schema migration.